lib/node_modules/@stdlib/blas/ext/base/wasm/dnansumkbn2/lib/module.js

```diff
@@ -0,0 +1,197 @@
+var WasmModule = require( '@stdlib/wasm/module-wrapper' );
+var format = require( '@stdlib/string/format' );
+var wasmBinary = require( './binary.js' );
+
+
+// MAIN //
+
+/**
+* BLAS routine WebAssembly module wrapper constructor.
+*
+* @constructor
+* @param {Object} memory - WebAssembly memory instance
+* @throws {TypeError} must provide a WebAssembly memory instance
+* @returns {Module} module instance
+*
+* @example
+* var Memory = require( '@stdlib/wasm/memory' );
+* var oneTo = require( '@stdlib/array/one-to' );
+*
+* // Create a new memory instance with an initial size of 10 pages (640KiB) and a maximum size of 100 pages (6.4MiB):
+* var mem = new Memory({
+*     'initial': 10,
+*     'maximum': 100
+* });
+*
+* // Create a BLAS routine:
+* var dnansumkbn2 = new Module( mem );
+* // returns <Module>
+*
+* // Initialize the routine:
+* dnansumkbn2.initializeSync();
+*
+* // Define a vector data type:
+* var dtype = 'float64';
+*
+* // Specify a vector length:
+* var N = 4;
+*
+* // Define a pointer (i.e., byte offset) for storing the input vector:
+* var xptr = 0;
+*
+* // Write vector values to module memory:
+* dnansumkbn2.write( xptr, [ 1, 2, 3, NaN ] );
+*
+* // Perform computation:
+* var sum = dnansumkbn2.main( N, xptr, 1 );
+* // returns 6.0
+*/
+function Module( memory ) {
+	if ( !( this instanceof Module ) ) {
+		return new Module( memory );
+	}
+	if ( !isWebAssemblyMemory( memory ) ) {
+		throw new TypeError( format( 'invalid argument. Must provide a WebAssembly memory instance. Value: `%s`.', memory ) );
+	}
+	// Call the parent constructor:
+	WasmModule.call( this, wasmBinary, memory, {
+		'env': {
+			'memory': memory
+		}
+	});
+
+	return this;
+}
+
+// Inherit from the parent constructor:
+inherits( Module, WasmModule );
+
+/**
+* Computes the sum of double-precision floating-point strided array elements, ignoring `NaN` values and using a second-order iterative Kahan–Babuška algorithm.
+*
+* @name main
+* @memberof Module.prototype
+* @readonly
+* @type {Function}
+* @param {PositiveInteger} N - number of indexed elements
+* @param {NonNegativeInteger} xptr - input array pointer (i.e., byte offset)
+* @param {integer} strideX - stride length
+* @returns {number} sum
+*
+* @example
+* var Memory = require( '@stdlib/wasm/memory' );
+*
+* // Create a new memory instance with an initial size of 10 pages (640KiB) and a maximum size of 100 pages (6.4MiB):
+* var mem = new Memory({
+*     'initial': 10,
+*     'maximum': 100
+* });
+*
+* // Create a BLAS routine:
+* var dnansumkbn2 = new Module( mem );
+* // returns <Module>
+*
+* // Initialize the routine:
+* dnansumkbn2.initializeSync();
+*
+* // Define a vector data type:
+* var dtype = 'float64';
+*
+* // Specify a vector length:
+* var N = 4;
+*
+* // Define a pointer (i.e., byte offset) for storing the input vector:
+* var xptr = 0;
+*
+* // Write vector values to module memory:
+* dnansumkbn2.write( xptr, [ 1, 2, 3, NaN ] );
```

Comment:
@ShabiShett07 You didn't update these examples. I suggest do another review of this PR to ensure you have updated all the necessary examples.